src/javascript/crypto/e2e/openpgp/pgpmime/mimenode.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview main module for the safemail app.
+ * @author yzhu@yahoo-inc.com (Yan Zhu)
+ */
+
+goog.provide('e2e.openpgp.pgpmime.MimeNode');
+
+goog.require('e2e.openpgp.pgpmime.Constants');
+goog.require('e2e.openpgp.pgpmime.Text');
```

Comment:
That's strange, for some reason it seems like three files weren't included (text.js, text_test.html, text_test.js). text.js is the file that was supposed to provide `pgpmime.Text`.
I'll push again.